hbase_sql/src/main/java/com/splicemachine/stream/StreamListener.java

```diff
@@ -73,21 +75,25 @@
     }
 
     public StreamListener(long limit, long offset, int batches, int batchSize) {
-        this(limit, offset, batches, batchSize, StreamableRDD.DEFAULT_PARALLEL_PARTITIONS, true);
+        this(limit, offset, batches, batchSize, StreamableRDD.DEFAULT_PARALLEL_PARTITIONS, true, true);
```

Comment:
Should the last two parameter be always true or configurable?

Reply:
The first and second constructors are used only in ITs with default parameters, I've added comments. Only the last one is used in the standard cases and gets parameters configured:

`public StreamListener(long limit, long offset, int batches, int batchSize, int parallelPartitions,
                          boolean throttleEnabled, boolean fileCacheEnabled)`